api/app/Models/User.php

```diff
@@ -594,22 +593,16 @@ public static function scopePublishingGroups(Builder $query, ?array $publishingG
     }
 
     /**
-     * Scope is IT
-     *
-     * Restrict a query by pool candidates that are for pools
-     * containing IT specific publishing groups
-     *
-     * @param  \Illuminate\Database\Eloquent\Builder  $query  The existing query being built
-     * @return \Illuminate\Database\Eloquent\Builder The resulting query
+     * Return users who have an available PoolCandidate in at least one IT pool.
      */
-    public static function scopeInITPublishingGroup(Builder $query)
+    public static function scopeAvailableInITPool(Builder $query): Builder
```

Comment:
Okay, I see now why you combined the two scopes! Could you rename this to `scopeAvailableInITPublishingGroup`? The distinction between IT Classification and IT Publishing group is confusing enough (as shown by me mistaking them several times in my review!) so lets be explicit about using publishing group.